docker/lnd/conf/lnd-bitcoind.conf

```diff
@@ -14,6 +14,7 @@ debuglevel=info
 maxpendingchannels=2
 nobootstrap=1
 tlsextradomain=lnd_btc
+minchansize=250000
```

Comment:
@treygriffith I was having trouble thinking of how often we would want these values to change. 

We could make this an ENV instead of throwing it into the `.conf` file so that we can edit on the fly. If that happened, then we would remove `min` and `max` channel balance from the configuration and just let LND handle the min/max which might be better for us in the long run, so it can be a dynamic number and fluctuate with the market.

We can punt this part for now, but wanted to get your thoughts